instant-xml-macros/src/lib.rs

```diff
@@ -156,10 +160,10 @@ impl<'a> Serializer {
 #[proc_macro_derive(ToXml, attributes(xml))]
 pub fn to_xml(input: TokenStream) -> TokenStream {
     let ast = parse_macro_input!(input as syn::DeriveInput);
-
     let ident = &ast.ident;
     let root_name = ident.to_string();
     let mut output: proc_macro2::TokenStream = TokenStream::from(quote!("".to_owned())).into();
```

Comment:
This way of creating a `proc_macro2::TokenStream` seems much more complex than needed. Also I think the code will look cleaner if you import `proc_macro2::TokenStream` as `TokenStream` and make explicitly qualify `proc_macro::TokenStream`.

Reply:
done